### PR TITLE
Courses

### PR DESCRIPTION
**Pull request by hapo-anhnd**

### CODE REVIEW

File: scss/style.scss [hunk 1]
```diff
@@ -172,6 +172,114 @@
...
+      }
+    }
+  }
+}
+
+.block-courses {
+  @extend %block;
+
+  margin-top: -100px; 
+}
```
Comment (gs-chinhnc) [inline scss/style.scss]: thừa line space

**Review by Hapo-congbv — APPROVED** [review]
LGTM

**Review by gs-chinhnc — APPROVED** [review]
LGTM

**Review by hapo-hungnn97 — APPROVED** [review]
LGTM

**Review by hapo-nghialuu — APPROVED** [review]
LGTM

**Pull request merged**